### DISCUSSION TITLE
環境構築についてのお問い合わせ

### DISCUSSION THREAD

**Discussion by y2klink** [cbi-society/cheminfotutorial1023#21] [Q&A]
講師の方へ  
吉田と申します。環境構築について質問させてください。  
conda activate cbi2023が正常に完了しない事象が発生しました。  
source activateを実行すると、仮想環境に入れたようなのですが、これでも問題ないでしょうか？  
```
yoshida@FCCLS0010041-00:~/cheminfotutorial1023$ conda activate cbi2023  
usage: conda [-h] [-v] [--no-plugins] [-V] COMMAND ...  
conda: error: argument COMMAND: invalid choice: 'activate' (choose from 'clean', 'compare', 'config', 'create', 'info', 'init', 'install', 'list', 'notices', 'package', 'remove', 'uninstall', 'rename', 'run', 'search', 'update', 'upgrade', 'content-trust', 'doctor', 'repoquery', 'env')
yoshida@FCCLS0010041-00:~/cheminfotutorial1023$ conda  info --envs  
# conda environments:  
#  
base                     /home/yoshida/miniconda3  
cbi2023                  /home/yoshida/miniconda3/envs/cbi2023  

yoshida@FCCLS0010041-00:~/cheminfotutorial1023$ conda activate cbi2023  
usage: conda [-h] [-v] [--no-plugins] [-V] COMMAND ...    
conda: error: argument COMMAND: invalid choice: 'activate' (choose from 'clean', 'compare', 'config', 'create', 'info', 'init', 'install', 'list', 'notices', 'package', 'remove', 'uninstall', 'rename', 'run', 'search', 'update', 'upgrade', 'content-trust', 'doctor', 'repoquery', 'env')  
yoshida@FCCLS0010041-00:~/cheminfotutorial1023$ source activate cbi2023  
(cbi2023) yoshida@FCCLS0010041-00:~/cheminfotutorial1023$  
```

**Comment by iwatobipen** [comment]
ご利用環境がわからないのでなんともいえませんが
こちらなど参考になりませんか？

https://community.anaconda.cloud/t/after-i-install-i-cannot-activate-the-enviroment-and-i-fixed-it-but-what-was-wrong/59968/2

https://japan-challenge.net/conda-activate%E3%81%AE%E3%82%A8%E3%83%A9%E3%83%BC%E3%82%92conda-init%E3%81%A7%E8%A7%A3%E6%B1%BA%E3%81%99%E3%82%8B/

**Reply by y2klink** [reply]
ご返信ありがとうございます。  
＞ご利用環境がわからないのでなんともいえませんが
失礼いたしました。windows10上にWSL(Ubuntu 22.04)で構築しています。  

＞こちらなど参考になりませんか？  
色々とやっていると、なんとかJupyterNotebookまでを起動させることが出来ました。  
ありがとうございます。

**Comment by kzfm** [comment]
吉田様 @y2klink 

source activateで仮想環境に入れたのであれば問題ないと思います。

- [参考](https://stackoverflow.com/questions/53995171/anaconda-conda-error-argument-command-invalid-choice-when-trying-to-update-pa)

**Reply by y2klink** [reply]
ご返信ありがとうございます。問題ないとのことで安心いたしました。
